stylus/ui-components/alerts.styl

```diff
@@ -30,6 +29,7 @@ $alerts
 
     .coz-alert
         display          block
+        z-index          100
```

Comment:
As far as I can tell, the other z-index in the file (`coz-alerter`) has no effect. To me it's a bugfix, but I'm not sure why the other z-index was there in the first place. I'm happy to change it back if it has implications I didn't see.